frontend/src/Components/Footer/Footer.style.ts

```diff
@@ -0,0 +1,172 @@
+import styled from "styled-components";
+import { Colors } from "../../Assets/styles/thems";
+
+export const Container = styled.footer`
+  position: relative;
+  background-color: ${Colors.pelorous};
+  width: 100vw;
+
+  .f-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+
+    padding-top: 8px;
+    padding-bottom: 6px;
+
+    padding-left: 4px;
+    padding-right: 4px;
+
+    @media (min-width: 1024px) {
+      margin-top: 40px;
+      margin-bottom: 40px;
+
+      padding-left: 112px;
+      padding-right: 112px;
+    }
```

Comment:
Mixing between  styled-components & classnames is possible, but it can make your code less readable.
I prefer you use styled-components instead of passing classnames.

So here for example, instead of passing the ".f-container" css rule for an element which has this classname, make that element a styled-component.
Apply this to all places in your code where you pass classnames.